store/sqlstore/emoji_store.go

```diff
@@ -100,26 +100,27 @@ func (es SqlEmojiStore) Get(id string, allowFromCache bool) (*model.Emoji, *mode
 	return emoji, nil
 }
 
-func (es SqlEmojiStore) GetByName(name string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var emoji *model.Emoji
+func (es SqlEmojiStore) GetByName(name string) (*model.Emoji, *model.AppError) {
 
-		if err := es.GetReplica().SelectOne(&emoji,
-			`SELECT
+	var emoji *model.Emoji
+
+	if err := es.GetReplica().SelectOne(&emoji,
+		`SELECT
 				*
```

Comment:
Please, keep the consistency of the indentation in the query string (just remove 1 tab on each line)